#!/usr/bin/python3
def raise_exception_msg(message=""):
    try:
        k = boy/7
    except NameError:
        print(message)
